common/views/components/TextInput/TextInput.js

```diff
@@ -34,21 +34,23 @@ type Props = {
   // We can also pass inputProps here
 }
 
-const TextInput = ({
+// $FlowFixMe
```

Comment:
I've been using `$FlowFixMe (forwardRef)` to help find and replace, but not sure we'll have much more.